src/blocks/card.scss

```
@@ -0,0 +1,83 @@
+.product-cards {
+  border-radius: 5px;
+  border: 1px solid $gray-elements;
+  background: $white;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+  &__tittle {
```

```suggestion
  }
  
  &__tittle {
```
add spaces between classes